### PR TITLE
Added 2 files by Sridhar

### PR DESCRIPTION
**Pull request by sridhar353**
Training stuff

### CODE REVIEW

**Comment by sridhar353** [comment]
#11

File: _posts/0000-01-29-sridhar353.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This is Line 1
+This is line 2
```
Comment (amanagrawal) [inline _posts/0000-01-29-sridhar353.md]: Image is missing

Comment (sridhar353) [reply _posts/0000-01-29-sridhar353.md]: That's right. I added a text file instead. Thanks for the comment

**Review by sridhar353 — COMMENTED** [review]
Ok, all comments reviewed and approved

**Review by sridhar353 — COMMENTED** [review]
Comment reviwed

**Review by beardofedu — APPROVED** [review]

**Comment by kornbert** [comment]
Great pull request!

File: _posts/0000-01-29-sridhar353.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a new line
```
Comment (kornbert) [inline _posts/0000-01-29-sridhar353.md:1]: @sridhar353 Excellent addition!

Comment (kornbert) [reply _posts/0000-01-29-sridhar353.md:1]: Very nice.

**Review by kornbert — COMMENTED** [review]
Very good!

**Pull request merged**